content/3. Procedurele-informatie/3. Processen/4. Implementatieproces/2. Bouwen softwaresysteem/While/1. While.md

```diff
@@ -0,0 +1,82 @@
+---
+title: 1. While
+taxonomie:
+  - ib-19.2.Controlestructuur-Herhaling-While.PI
+  - ib-19.3.Controlestructuur-Herhaling-While.PI
+---
+
+> Meer informatie over [[content/2. Ondersteunende-informatie/3. Processen/4. Implementatieproces/2. Bouwen softwaresysteem/While/1. While|While]]
+
+## Stappen in de Systematische Probleem Aanpak
+### 1. Doel
+Bepaal waarom een _while_-loop nodig is. Bijvoorbeeld: Het programma moet een taak herhalen totdat een specifieke voorwaarde wordt voldaan.
```

Comment:
Hier heb je while onderstreept en OI dikgedrukt. Ik weer het zo niet uit mijn hoofd en pc niet bij de hand, maar check even of / dat dit consistent is